content/docs/faq.md

```diff
@@ -69,15 +73,13 @@ Yes. Scratch Addons should not have any security issues in its most recent versi
 
 ### How can I report a security vulnerability?
 
-If you happen to find a security vulnerability, please contact World_Languages privately by emailing `worldxlanguages (at) gmail.com`. If you don't get a response within 48 hours, please [create an issue](https://github.com/ScratchAddons/ScratchAddons/issues/) mentioning that you had sent an email.
-
-You can either [read our security policy](https://github.com/ScratchAddons/ScratchAddons/security/policy) or [check our advisories that we have published](https://github.com/ScratchAddons/ScratchAddons/security/advisories?state=published).
+Please report security vulnerabilities privately to us. See our [security policy](https://scratchaddons.com/docs/policies/security/) for more information, such as contact details and the historical list of reported vulnerabilities.
 
 ### Will my account be safe when using Scratch Addons?
 
 Yes. Features related to your Scratch account are thoroughly reviewed by Scratch Addons contributors to ensure that they will not harm your Scratch account. Scratch Addons keeps your account credentials secure, and it will not modify or delete your projects or assets without your permission. However, you may use the extension without any account-related features if you so choose.
 
-Addons on Scratch Addons also have been audited by multiple contributors on the repository, so no-one can just slip some malicious code under our eyes.
+Addons on Scratch Addons also have been audited by multiple contributors on the repository, so no one can just slip some malicious code under our eyes.
```

Comment:
I prefer "nobody" over "no one" but that's more of a personal preference

Reply:
Controversial. I suggest you open a new PR after this one gets merged if you think it's worth it.